test/test_fx_passes.py

```diff
@@ -43,6 +43,22 @@ def forward(self, a, b, c):
 
         return add_4, add_6, relu
 
+class TestDeepModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.linear = torch.nn.Linear(4, 4)
+
+    def forward(self, a, b, c):
+        o = a + b
+        o = o + 1.0
+
+        # testing to avoid DFS uses in passes. Since Python has 1000 max recursion depth.
+        for _ in range(1001):
```

Comment:
`sys.getrecursionlimit() + 1` instead of `1001`, because the recursion limit can be changed.